pkg/alert/alertconfigmap.go

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright (C) 2018 Synopsys, Inc.
+
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements. See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownershia. The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License. You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied. See the License for the
+specific language governing permissions and limitations
+under the License.
+*/
+
+package alert
+
+import (
+	"strings"
+
+	horizonapi "github.com/blackducksoftware/horizon/pkg/api"
+	"github.com/blackducksoftware/horizon/pkg/components"
+	log "github.com/sirupsen/logrus"
+)
+
+// getAlertConfigMap returns a new ConfigMap for an Alert
+func (a *SpecConfig) getAlertConfigMap() *components.ConfigMap {
+	configMap := components.NewConfigMap(horizonapi.ConfigMapConfig{
+		Name:      "blackduck-alert-config",
+		Namespace: a.config.Namespace,
+	})
+
+	// Add Environs
+	configMapData := map[string]string{}
+	for _, environ := range a.config.Environs {
+		vals := strings.Split(environ, ":")
+		if len(vals) != 2 {
+			log.Errorf("Could not split environ '%s' on ':'", environ)
+			continue
+		}
+		environKey := strings.Trim(vals[0], " ") // trim removes leading/trailing " "
```

Comment:
Apparently, there's also a [TrimSpace](https://golang.org/pkg/strings/#TrimSpace), which may increase readability.

Reply:
I made the change to TrimSpace